opencga-core/src/main/java/org/opencb/opencga/core/api/ParamConstants.java

```diff
@@ -570,6 +570,12 @@ public class ParamConstants {
     public static final String JOB_TAGS = "jobTags";
     public static final String JOB_TAGS_PARAM = "tags";
     public static final String JOB_TAGS_DESCRIPTION = "Job tags";
+    public static final String JOB_SCHEDULED_START_TIME_DESCRIPTION = FieldConstants.JOB_SCHEDULED_START_TIME_DESCRIPTION;
+    public static final String JOB_SCHEDULED_START_TIME = FieldConstants.JOB_SCHEDULED_START_TIME;
```

Comment:
Existing extra "job" params for generic job execution were prefixed with "job", like "jobTags" or "jobDependsOn"
Why these new "dryRun" , "scheduledStartTime" and "proprity"  are not?

Reply:
Done